scripts/install-worker.sh

```diff
@@ -502,13 +502,13 @@ sudo yum install -y amazon-ssm-agent
 ################################################################################
 
 BASE_AMI_ID=$(imds /latest/meta-data/ami-id)
-cat << EOF > /tmp/release
```

Comment:
these will change too

Reply:
Yeah, anything that was using `/tmp` should now be using the configured working directory. In this case, we just create a file in a temp location and then move it into the final location, so it doesn't matter where the temp location is.